test_driver/no_driver.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:crowdleague/models/app/app_state.dart';
+import 'package:crowdleague/utils/redux/services_bundle.dart';
+import 'package:crowdleague/utils/redux/store_operation.dart';
+import 'package:crowdleague/widgets/crowd_league_app.dart';
+import 'package:flutter/material.dart';
+import 'package:redux_remote_devtools/redux_remote_devtools.dart';
+
+Future<void> main() async {
+  // enableFlutterDriverExtension() was removed to allow us to manually
```

Comment:
this file is just `test_driver/firebase_emulators.dart` with the first line (`enableFlutterDriverExtension();`) removed.